src/Microsoft.DocAsCode.Build.JavaScriptReference/BuildInputs/PageViewModel.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.JavaScriptReference
+{
+    using System;
+    using System.Collections.Generic;
+
+    using Newtonsoft.Json;
+    using YamlDotNet.Serialization;
+
+    using Microsoft.DocAsCode.DataContracts.Common;
+    using Microsoft.DocAsCode.YamlSerialization;
+
+    [Serializable]
+    public class PageViewModel
+    {
+        [YamlMember(Alias = "items")]
+        [JsonProperty("items")]
+        public List<ItemViewModel> Items { get; set; } = new List<ItemViewModel>();
+
+        [YamlMember(Alias = "references")]
+        [JsonProperty("references")]
+        public List<ReferenceViewModel> References { get; set; } = new List<ReferenceViewModel>();
+
+        [YamlMember(Alias = "shouldSkipMarkup")]
+        [JsonProperty("shouldSkipMarkup")]
+        public bool ShouldSkipMarkup { get; set; }
```

Comment:
JSDoc has the a option to choose whether to markup text. This property may be used later.

---
In reply to: [93179333](https://github.com/dotnet/docfx/pull/1016#discussion_r93179333) [](ancestors = 93179333)